src/inputs/input.stories.html

```diff
@@ -0,0 +1,257 @@
+<article>
+  <h2 class="title">Inputs</h2>
+  <p class="title">
+    There are 3 types of input fields defined in the styleguide:
+  </p>
+  <div class="container">
+    <div>
+      <h3 class="title">Underlined Inputs</h3>
+      <form>
+        <div class="omrs-input-group">
+          <input class="omrs-input-underlined" type="text" required />
+          <label>Name</label>
+          <span class="omrs-input-helper">Helper Text</span>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+          >
+            <path fill="none" d="M0 0h24v24H0V0z" />
+            <circle cx="15.5" cy="9.5" r="1.5" />
+            <circle cx="8.5" cy="9.5" r="1.5" />
+            <path
+              d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8zm-5-6c.78 2.34 2.72 4 5 4s4.22-1.66 5-4H7z"
+            />
+          </svg>
+        </div>
+        <div class="omrs-input-group">
+          <input
+            class="omrs-input-underlined omrs-input-danger"
+            type="text"
+            required
+          />
+          <label>Tel</label>
```

Comment:
This HTML is inaccessible to screen readers. The screen reader won't know that this `<label>` applies to this `<input>`.

See https://kentcdodds.com/blog/please-stop-building-inaccessible-forms-and-how-to-fix-them for options to fix.

Reply:
I think I maybe lean towards adding `for` attributes and `id` attributes. But am open to the other solutions that Kent suggests in that blog post. What do you think?

Reply:
Since the behavior of the label relies very much on the input e.g on focus, it is might be very difficult to achieve the same behavior if we move the label up to be on top of the input without using some JavaScript i.e 
```
<label for="name>Name</label>
<input class="omrs-input-outlined" id="name" />
```

However, I gave this a shot and it seemed to work alright with the accessibility voiceover :
```
<input class="omrs-input-outlined" id="name" />
<label for="name>Name</label>
```

Another option I can think of might be to define the html contract like this

```
<label class="omrs-input-outlined">
  <input>
  <span>Name</span>
</label>
```

What would you suggest?

Reply:
> Since the behavior of the label relies very much on the input e.g on focus, it is might be very difficult to achieve the same behavior if we move the label up to be on top of the input without using some JavaScript 

Could you help me understand this? I am unaware of any differences in accessibility when the `<label>` is before or after the `<input>`. Is that what you're saying? That a `<label>` before the `<input>` doesn't work with screen readers? Or am I misunderstanding?

Reply:
That last option you suggested would make it pretty difficult to implement the vertical input form layout that I mentioned in my big comment at the top ^. I think I prefer using either `<label for="">` or `<label id="hi">` with `<input aria-labelledby="hi">`

Reply:
makes sense. With regards to the big comment at the top about horizontally aligned forms, I think it may be possible to achieve that with the current html css contract using 
`display: flex` and the `order` property to place the label on the left of the input.  I'm thinking we'd have to replace `.omrs-input-group` with `.omrs-input-group-horizontal` to achieve that.

Reply:
Pushed new changes according to the review @joeldenning